__tests__/app.test.js

```diff
@@ -137,3 +137,51 @@ describe("/api/articles/:article_id", () => {
     });
   });
 });
+describe("/api/articles", () => {
+  describe("GET", () => {
+    test("200: Responds with array of article objects with its properties", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          expect(Array.isArray(body.articles)).toBe(true);
```

Comment:
147 - We don't techncially need this as we are using  a forEach on line 149 which implies that body.articles is an array